runner/runner.go

```diff
@@ -392,7 +393,7 @@ type StartChannels struct {
 	KillChan chan bool
 }
 
-func Start(manifest *manifest.Manifest, reporter reporter.Reporter, killChannel chan bool, debug bool) (Runner, error) {
+func Start(manifest *manifest.Manifest, reporter io.Writer, killChannel chan bool, debug bool) (Runner, error) {
```

Comment:
rename `reporter` to something meaningful, since it is no longer `reporter.Reporter`